### ISSUE TITLE
SAINT model has incorrect defaults

### ISSUE THREAD

**Issue by Xemorr** [pykt-team/pykt-toolkit#166]
According to the SAINT [paper](https://arxiv.org/pdf/2002.07033.pdf), the default dropout rate is 0.1. PyKT has it set to 0.2 by default

**Comment by lycyhrc** [comment]
> According to the SAINT [paper](https://arxiv.org/pdf/2002.07033.pdf), the default dropout rate is 0.1. PyKT has it set to 0.2 by default

Because we need to search for the best superparameter in wandb, PYKT in yaml file (https://github.com/pykt-team/pykt-toolkit/blob/main/examples/seedwandb/saint.yaml) Lines 21-22 set different dropout values for search.
Therefore, the dropout value of examples/wandb_saint_train.py can be freely adjusted to suit your needs. @Xemorr

**Comment by Xemorr** [comment]
Yes, I know it can be freely adjusted but I think the **default** should be the one in the paper.
